src/encoded/static/components/item-pages/HiGlassViewConfigView.js

```diff
@@ -312,7 +318,7 @@ export class HiGlassViewConfigTabView extends React.PureComponent {
      */
     handleUpdateAllTilesets(evt) {
         evt.preventDefault();
-        const { updatedTilesetField, trackInfo } = this.state;
+        const { updatedTilesetField, trackInfo, filesTableSearchHref } = this.state;
```

Comment:
Codacy found an issue: ['filesTableSearchHref' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6090401)